docker_services_cli/cli.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Docker-Services-CLI is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CLI module."""
+
+import sys
+from distutils.sysconfig import get_python_lib
+from pathlib import Path
+
+import click
+
+from .env import set_env
+from .services import services_down, services_up
+
+
+@click.group()
+@click.version_option()
+def cli():
+    """Initialize CLI context."""
+
+
+@cli.command()
+@click.argument(
+    "action",
+    default="up",
+    required=False,
+    type=click.Choice(["up", "down"], case_sensitive=False),
+)
+@click.argument("services", nargs=-1, required=False)  # -1 incompat with default
+@click.option(
+    "--filepath",
+    "-f",
+    required=False,
+    default=f"{get_python_lib()}/docker_services_cli/docker-services.yml",
+    type=click.Path(exists=True),
+    help="Path to a docker compose file with the desired services definition.",
+)
+def services(action, services, filepath):
+    """Boots up or down the required services."""
+    set_env()
```

Comment:
Not to be fixed now, but if you agree we can create an issue to fix it in the future.

In general it is not good practice to change the environment variables of a user's shell. Therefore, I would say we need an `unset_env` to clean whatever environment variables have been created/modified in the user's environment.

Reply:
IIRC, the variables only live while the python script executes. Therefore after CLI exit (e.g. when up finishes) they are unset.
```
(docker-services) ~/Workspace/inveniosw/docker-services(master ✔) ENV | grep ES
TERM_SESSION_ID=...
ITERM_SESSION_ID=...
LESS=-R
(docker-services) ~/Workspace/inveniosw/docker-services(master ✔) docker-services-cli services up es
Creating network "docker_services_cli_default" with the default driver
Creating docker_services_cli_es_1 ... done
(docker-services) ~/Workspace/inveniosw/docker-services(master ✔) ENV | grep ES
TERM_SESSION_ID=...
ITERM_SESSION_ID=...
LESS=-R
```

If they would live, after running the command I should get `ES_VERSION` plus all the `*_LATEST`.

Reply:
Perfect